qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -1280,63 +1280,115 @@
         "enabled": false,
         "text": "Publication Discovery"
       },
+      "exportToWorkspace": {
```

Comment:
are all these gitops.json changes necessary?

Reply:
Yes, these changes are for the discovery portal page and I have followed the configuration from bloodPAC and Heal's gitops.json. They are required to show appropriate fields in the UI, populate the customized search filter, and show details.